api/data_test.go

```diff
@@ -42,7 +42,7 @@ func TestGetCacheTimeEndpoint(t *testing.T) {
 				}
 			},
 		}
-		dataStoreAPI := setupAPIWithStore(ctx, dataStoreMock)
+		dataStoreAPI := setupAPIWithStore(ctx, true, dataStoreMock)
```

Comment:
Just for readability I would declare this parameter isPublishing in a variable and pass the variable. If you just pas "true" you might wonder what parameter you passing here.